apps/nextjs/src/pages/index.tsx

```diff
@@ -3,7 +3,7 @@ import Head from "next/head";
 import Link from "next/link";
 import { LandingPageHeaderMenu } from "../components/HeaderMenu";
 
-const LandingPage: NextPage = () => {
+const LandingPage = () => {
```

Comment:
Why take away the next page declaration?

Reply:
Added back in commit https://github.com/Pickle-App/pickle-app/pull/8/commits/260068aaa3acb14b1c39ee9599486a41e31288a1